### PR TITLE
feat(file upload): update file size, network and file type error messages

### PR DESCRIPTION
**Pull request by dat-gcio**
## Description
Update platform file upload component's error messages.

## Testing done
- [ ] manual
- [ ] unit

## Screenshots
mult_error
network_error

## Acceptance criteria
- [ ] Update `Network request failed` 
to `We're sorry. We had a connection problem. Please delete the file and try again.`
- [ ] Update `File is too large to be uploaded`
to `We couldn't upload your file because it's too big. Please delete this file. Then upload a file that's {fileSize} or less.`
- [ ] Update `File is too small to be uploaded`
to `We couldn't upload your file because it's too small. Please delete this file. Then upload a file that's {fileSize} or more.`
- [ ] Update `File is not one of the allowed types`
to `We couldn't upload your file because we can't accept this type of file. Please delete the file. Then try again with a {fileType(s)} file.`

## Definition of done
- [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25405)
- [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25320)

### CODE REVIEW

File: src/platform/forms-system/src/js/actions.js
Comment (bkjohnson) [inline src/platform/forms-system/src/js/actions.js:237]: Has someone from content reviewed these message changes? cc @peggygannon

Comment (dat-gcio) [reply src/platform/forms-system/src/js/actions.js:237]: I believe @DanielleThierryUSDSVA  worked on this with our designer Dene.

Comment (DanielleThierryUSDSVA) [reply src/platform/forms-system/src/js/actions.js:237]: I did work on these with Dené, yes. Thanks, @dat-gcio. 

@peggygannon, here's the content we worked out. Let me know if you have any questions. I'll also ask Randi to update in the content style guide. Thanks!

**Network request failed**

We're sorry. We had a connection problem. Please delete the file and try again.
[Delete file]

**File too large**
We couldn't upload your file because it's too big. Please delete this file. Then upload a file that's 10MB or less.
[Delete file]

**File too small**
We couldn’t upload your file because it’s too small. Please delete this file. Then upload a file that’s 1B or more.
[Delete file]

**File not allowed type**
We couldn't upload your file because we can't accept this type of file. Please delete the file. Then try again with a .pdf, .jpeg, .jpg, or .png file.
[Delete file]

Comment (peggygannon) [reply src/platform/forms-system/src/js/actions.js:237]: @DanielleSoCompany OK, thank you for the update.

Comment (dat-gcio) [reply src/platform/forms-system/src/js/actions.js:237]: @peggygannon @DanielleThierryUSDSVA Are we good to move forward, or do we need to wait on additional approval?

**Review by peggygannon — COMMENTED** [review]
@bkjohnson @dat-gcio @DanielleThierryUSDSVA Updates look good to me. thank you!

**Comment by DanielleThierryUSDSVA** [comment]
Thanks, @peggygannon! @dat-gcio we're good to go on the content front. Thanks!

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**
